### PR TITLE
Increase the accessibility of my code

### PR DESCRIPTION
**Pull request by MeqdamAlqudah**
1-Page titles-changes:-
https://github.com/MeqdamAlqudah/Portfolio-setup-and-mobile-version-skeleton/blob/63bf6f748b9d536512e4c62a33dc64312fa21bbc/index.html#L32
2-Image text alternatives** (alt text):- (I changed most of the img elements in this link)
https://github.com/MeqdamAlqudah/Portfolio-setup-and-mobile-version-skeleton/blob/63bf6f748b9d536512e4c62a33dc64312fa21bbc/index.html#L37-L374
3-Text headings-no issues found
4-Color contrast:-
a-https://github.com/MeqdamAlqudah/Portfolio-setup-and-mobile-version-skeleton/blob/63bf6f748b9d536512e4c62a33dc64312fa21bbc/style.css#L358-L367
b-https://github.com/MeqdamAlqudah/Portfolio-setup-and-mobile-version-skeleton/blob/63bf6f748b9d536512e4c62a33dc64312fa21bbc/style.css#L385-L395
c-https://github.com/MeqdamAlqudah/Portfolio-setup-and-mobile-version-skeleton/blob/63bf6f748b9d536512e4c62a33dc64312fa21bbc/style.css#L403-L412
5-Resize- no issues found
6-Interaction-no issues found
7-Moving content - no issues found
8-Multimedia - no issues found
9-The basic structure of the page:-
a-https://github.com/MeqdamAlqudah/Portfolio-setup-and-mobile-version-skeleton/blob/63bf6f748b9d536512e4c62a33dc64312fa21bbc/index.html#L287
b-https://github.com/MeqdamAlqudah/Portfolio-setup-and-mobile-version-skeleton/blob/63bf6f748b9d536512e4c62a33dc64312fa21bbc/index.html#L300
c-https://github.com/MeqdamAlqudah/Portfolio-setup-and-mobile-version-skeleton/blob/63bf6f748b9d536512e4c62a33dc64312fa21bbc/index.html#L314

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -18,12 +18,23 @@
...
+<script>
+  window.dataLayer = window.dataLayer || [];
+  function gtag(){window.dataLayer.push(arguments);}
+  gtag('js', new Date());
+
+  gtag('config', 'GA_MEASUREMENT_ID');
+  gtag('config', 'GA_MEASUREMENT_ID', { 'send_page_view': false });
+
+</script>
     <title>Portifolio</title>
```
Comment (Mihndim2020) [inline index.html:32]: - [ ] To make your page more accessible, your title should adequately and briefly describes the content of the page. Please kindly fix that. 

youmi

**Review by Mihndim2020 — CHANGES_REQUESTED** [review]
### Hi @, 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Descriptive Readme file
:heavy_check_mark: Good job passing linter checks

There are some issues that you still need to work on to get this project approved, but you are almost there!
ALMOST-MINION


# Status: Requires Changes ♻️
_Check the comments below and  under the review._

- [ ] Please as it is a requirement for this project, edit you pull request to include the information of the screenshot below. 

meq

- [ ] Please include all the points in the screenshot above as you add the report to the description of your pull request. 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -18,12 +18,23 @@
       href="https://fonts.googleapis.com/css2?family=Playfair+Display&display=swap"
       rel="stylesheet"
     />
-    <title>Portifolio</title>
+    <!-- Global site tag (gtag.js) - Google Analytics -->
```
Comment (D4nielJ) [inline index.html:21]: - [ ] Please make sure to follow all of the requirements asked by the last reviewer. These include updating the description of this pull request to include a report on every one of the accessibility checks you've done and links to the lines where you made changes to improve it.

If you unsure what this means, make sure to read carefully [the requirements of this milestone here.](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/6_accessability.md)

image

**Review by D4nielJ — CHANGES_REQUESTED** [review]
## Hello @MeqdamAlqudah, 

You're doing great! 👏 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

## Happy coding and stay awesome! 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can always find me on slack!

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (MeqdamAlqudah) [reply index.html:21]: I change it this is the index.html file for deploy not the original code my index.html file have the title Meqdam's portfolio

**Review by VanessaAoki — APPROVED** [review]
# STATUS: APPROVED 🟢 
Hi @MeqdamAlqudah , 

Your project is complete!  There is nothing else to say other than... it's time to merge it! 
Congratulations! 🎉 

approved

## Project highlights:
- [x] All linter checks are passing :heavy_check_mark: 
- [x] Your README file looks very professional and the PR has a descriptive title/accessibility report :memo: 
- [x] Descriptive commits and correct use of the GitHub Flow :fire: 
- [x] Your code passed the following checks: :white_check_mark: page title :white_check_mark: image text alternatives :white_check_mark: text headings :white_check_mark: color contrast :white_check_mark: rezise :white_check_mark: moving content :white_check_mark: multimedia :white_check_mark: basic structure of the page


Cheers and Happy coding!👏👏👏


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**